{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": []
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "cve@mitre.org",
      "ID": "CVE-2019-20428"
    },
    "affects": {
      "vendor": {
        "vendor_data": []
      }
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "In the Lustre file system before 2.12.3, the ptlrpc module has an out-of-bounds read and panic due to the lack of validation for specific fields of packets sent by a client. The ldl_request_cancel function mishandles a large lock_count parameter."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": []
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "http://lustre.org/",
          "refsource": "MISC",
          "tags": [],
          "url": "http://lustre.org/"
        },
        {
          "name": "http://wiki.lustre.org/Lustre_2.12.3_Changelog",
          "refsource": "MISC",
          "tags": [],
          "url": "http://wiki.lustre.org/Lustre_2.12.3_Changelog"
        },
        {
          "name": "https://jira.whamcloud.com/browse/LU-12603",
          "refsource": "MISC",
          "tags": [],
          "url": "https://jira.whamcloud.com/browse/LU-12603"
        },
        {
          "name": "https://review.whamcloud.com/#/c/36108/",
          "refsource": "MISC",
          "tags": [],
          "url": "https://review.whamcloud.com/#/c/36108/"
        }
      ]
    }
  },
  "impact": {},
  "lastModifiedDate": "2020-01-27T12:47Z",
  "publishedDate": "2020-01-27T05:15Z"
}